src/main/python/car_trecweb.py

```diff
@@ -20,22 +25,48 @@ def writer(p, fp):
     content += (u'<DOCHDR>\n')
     content += (u'\n')
     content += (u'</DOCHDR>\n')
+    if meta_dict and p.para_id in meta_dict:
+        content += (u'<TITLE>\n')
+        content += (meta_dict[p.para_id]["title"])
+        content += (u'\n</TITLE>\n')
+        content += (u'<TAG>\n')

```

Comment:
I think we can do better with a more descriptive name. I think this is "Header" or "Section Heading" and so a tag that is appropriate.   It could also be combined with the title perhaps.

Reply:
I'll change to:
```
<HEADER>
Johannisthal Air Field
</HEADER >
```
is this what you mean?